### ISSUE TITLE
Suggestion - Access to Admin Panel Specifics for Judges

### ISSUE THREAD

**Issue by sleighwhite** [InZidiuZ/op-framework-issue-tracker#3022]
### Assurance

I have read the `README.md`

### Summary

Looking for some in-character command or maybe something in the comp for Judges to be able to check phone number ownership and also Twitter ownership without having to access the actual Admin Panel. 

For Twitter, I need: 

[FULL NAME]
[TWITTER USERNAME]
[THEIR DISCORD, OR "N/A"]
[PHONE NUMBER]
[GENDER]

For Phone Numbers, I need:

[Character ID}
Phone Number:**
**Full Name:**
**Date of Birth:**

### Reason

Currently, you can only access this information through the Admin Panel. Not all judges are staff with access to this, so it means we have to reach out to other members of the DOJ that are staff, but that sometimes slows things down for PD getting important info that is time-sensitive for roleplay. Or I have to reach out to other staff members that are unrelated to the situation in general which could potentially, unfortunately, open things up for meta-gaming. It would be really nice to just be able to handle this myself without having to go to others so I can expedite things for PD/FIB quickly and efficiently in my new role! :)

**Comment by ImArteria** [comment]
+1 on Suggestion, I think an in game server command would be best and I dont think would be difficult to implement. 
We can see twitter information via the twitter on the panel
Phone numbers/owners I think through the raspberry
Than home owners potentially through the in game commands?

**Comment by coalaura** [comment]
Idk, its a very powerful thing. I'd rather have it have to go through staff tbh

**Comment by sleighwhite** [comment]
Totally understand - just for clarification, not looking for access to the entire admin panel, just the specifics Judges can see anyway. It'd only be available to people with the job which at the minute is like 3 or 4 people.

**Comment by coalaura** [comment]
Yeah i meant number -> owner or twitter -> owner are pretty powerful things that can easily be abused or overused

**Comment by ImArteria** [comment]
I mean number can already be accessed via a raspberry right? Like sleigh said only 3-4 people having this, who 9/10 times are staff, it'd be easier without use of a panel for the 1/10 people who arent staff

**Comment by sleighwhite** [comment]
I think the thing is here that yes number/twitter can be abused easily, but I get that information anyway, right? If you're a judge, you're pretty trusted, I'd assume. I know I had to make several OOC assurances about how I might use/not use information I am privy to both IC and OOC. 

As it stands, I may have to reach out to a staff member and say "hey I have a subpoena that says I can have this information" but I can't show them the subpoena because it may have sensitive RP information on it that only judges/FIB are privy to. So they really have no way to know if I'm abusing it or not, right?

If it's too much of a hassle, that's understandable. Just thought this might help expedite some stuff within the DOJ that we've had some issues with in the past!

**Comment by coalaura** [comment]
signed by the state of San andreas so it cant be forged or edited

![yes](https://i.shrt.day/GIJESIto87.png)

**Comment by sleighwhite** [comment]
This is so fucking sick, thank you Laura!
